packages/react-magma-dom/src/components/Pagination/SimplePagination.tsx

```diff
@@ -0,0 +1,175 @@
+    font-weight: 500;
+  }
+`;
+
+const StyledNativeSelect = styled(NativeSelect)<{
+  disabled?: boolean;
+}>`
+  select {
+    color: ${props => (props.disabled ? props.theme.colors.neutral500 : '')};
+    cursor: ${props => (props.disabled ? 'not-allowed' : '')};
+  }
+  min-width: 87px;
+`;
+
+export const SimplePagination = React.forwardRef<
+  HTMLDivElement,
+  PaginationProps
+>((props, ref) => {
+  const {
+    count,
+    defaultPage,
+    disabled,
+    hideNextButton,
+    hidePreviousButton,
+    id: defaultId,
+    isInverse,
+    size = PageButtonSize.medium,
+    testId,
+    onPageChange,
+    ...other
+  } = props;
+  const theme = React.useContext(ThemeContext);
+  const buttonSize =
+    size === PageButtonSize.large ? ButtonSize.large : ButtonSize.medium;
+
+  const i18n = React.useContext(I18nContext);
+
+  const id = useGenerateId(defaultId);
+
+  let [selectedValue, setSelectedValue] = React.useState(
+    defaultPage ? defaultPage : 1
+  );
+
+  function handleChange(event) {
+    props.onChange &&
+      typeof props.onChange === 'function' &&
+      props.onChange(event);
+    setSelectedValue(event.target.value);
+  }
+
+  //Decreases number by one on previous button click
+  function handlePrev() {
+    if (selectedValue > 1) {
+      setSelectedValue(selectedValue - 1);
+    }
+  }
+
+  //Increases number by one on next button click
+  function handleNext() {
+    if (selectedValue < count) {
+      setSelectedValue(++selectedValue);
+    }
+  }
+
+  return (
+    <StyledWrapper
+      {...other}
+      theme={theme}
+      data-testid={testId}
+      disabled={disabled}
+      isInverse={isInverse}
+      ref={ref}
+    >
+      {!hidePreviousButton && (
+        <Tooltip content={i18n.pagination.previousButtonLabel}>
+          <NavButton
+            aria-label={i18n.pagination.previousButtonLabel}
+            variant={ButtonVariant.link}
+            color={ButtonColor.secondary}
+            disabled={selectedValue < 2 ? true : disabled}
+            icon={<ArrowBackIcon />}
+            isInverse={isInverse}
+            theme={theme}
+            onClick={handlePrev}
+            shape={ButtonShape.round}
+            size={buttonSize}
+          />
+        </Tooltip>
+      )}
+      <Spacer size={14} />
+
+      <StyledNativeSelect
+        aria-label={i18n.select.placeholder}
+        data-testid={`${testId}-select`}
+        disabled={disabled}
+        fieldId={id}
+        isInverse={isInverse}
+        onChange={handleChange}
+        theme={theme}
+        value={selectedValue}
+      >
+        {Array.from({ length: count }, (_, i) => (
+          <option
+            data-testid={`${testId}-option`}
+            key={i + count}
+            onChange={handleChange}
+            value={i + 1}
+          >
+            {i + 1}
+          </option>
+        ))}
+      </StyledNativeSelect>
+      <Spacer size={8} />
+      <label aria-hidden="true">of {count} pages</label>
```

Comment:
We should i18n these. Also, we need to account for when `count` is 1, it should say `page`